submission_runner.py

```diff
@@ -263,13 +262,6 @@ def train_once(
             rng=update_rng)
     except spec.TrainingCompleteError:
       training_complete = True
-    except RuntimeError as e:
-      if "out of memory" in str(e):
-        logging.warning(
-            f'error: GPU out of memory during training during step {global_step}, error: {str(e)}'  # pylint: disable=line-too-long
-        )
-        if torch.cuda.is_available():
-          torch.cuda.empty_cache()
```

Comment:
should we keep this?

Reply:
it's extra code that doesn't really catch OOM errors, I included it last time in hope that it might catch a real GPU OOM error